packages/pg/lib/client.js

```diff
@@ -20,6 +20,7 @@ class Client extends EventEmitter {
     this.database = this.connectionParameters.database
     this.port = this.connectionParameters.port
     this.host = this.connectionParameters.host
+    this.enableChannelBinding = false // set true to use SCRAM-SHA-256-PLUS when offered
```

Comment:
Should read this from `config` (which is also the answer to the question of how to configure this when using a `Pool`).

```suggestion
    this.enableChannelBinding = Boolean(config.enableChannelBinding) // set true to use SCRAM-SHA-256-PLUS when offered
```

Reply:
Sounds good.